jetcd-core/src/test/java/io/etcd/jetcd/WatchUnitTest.java

```diff
@@ -321,28 +320,28 @@ public void testWatcherCreateOnCancellationWithNoReason() throws InterruptedExce
     CountDownLatch latch = new CountDownLatch(1);
     AtomicReference<Throwable> ref = new AtomicReference<>();
     Watch.Listener listener = Watch.listener(
-      r -> {},
-      t -> {
-        ref.set(t);
-        latch.countDown();
-      }
+        r -> { },
+        t -> {
+          ref.set(t);
+          latch.countDown();
+        }
     );
 
     try (Watch.Watcher watcher = watchClient.watch(KEY, listener)) {
       WatchResponse createdResponse = createWatchResponse(0);
       responseObserverRef.get().onNext(createdResponse);
 
       WatchResponse canceledResponse = WatchResponse
-              .newBuilder()
-              .setCanceled(true)
-              .build();
+          .newBuilder()
```

Comment:
is this auto formatted? not sure why it has 4 spaces as the indent.

Reply:
It's formatted according to styleguide. Weird thing is that linter complains if indentation is less that what it should be, but not when it's more.

Reply:
i see.